repro.in

```diff
@@ -369,6 +369,7 @@ for rev in \$(git rev-list --all -- repos/); do
         exit 0
     fi
 done
+echo "ERROR: Failed to find commit this was built with (PKGBUILD checksum didn't match any commit)"
```

Comment:
Small nitpick, should this be send to stderr and not stdout?

Reply:
Good point, done!